locale/nl/LC_MESSAGES/rclone-manager.po

```diff
@@ -0,0 +1,199 @@
+# Spanish translation for gnome-shell-extension-rclone-manager.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the gnome-shell-extension-rclone-manager package.
+#
+# Heimen Stoffels <vistausss@fastmail.com>, 2022.
+msgid ""
+msgstr ""
+"Project-Id-Version: gnome-shell-extension-rclone-manager\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-04-12 13:20+0300\n"
+"PO-Revision-Date: 2022-05-09 17:10+0200\n"
+"Last-Translator: Heimen Stoffels <vistausss@fastmail.com>\n"
+"Language-Team: Dutch <kde-i18n-nl@kde.org>\n"
+"Language: nl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Lokalize 22.04.0\n"
+
+msgid " was already mounted"
+msgstr " is al aangekoppeld"
+
+msgid "Add config"
+msgstr "Configuratie toevoegen"
+
+msgid "Settings"
+msgstr "Voorkeuren"
+
+msgid "Watch"
+msgstr "Bijhouden"
+
+msgid "Unwatch"
+msgstr "Niet meer bijhouden"
+
+msgid "Mount"
+msgstr "Aankoppelen"
+
+msgid "Umount"
+msgstr "Ontkoppelen"
+
+msgid "Open"
+msgstr "Openen"
+
+msgid "Reconnect"
+msgstr "Opnieuw verbinden"
+
+msgid "Sync"
+msgstr "Synchroniseren"
+
+msgid "Delete"
+msgstr "Verwijderen"
+
+msgid "Error"
+msgstr "Foutmelding"
+
+msgid "Log"
+msgstr "Logboek"
+
+msgid "Disengage"
+msgstr "Ontkoppelen"
+
+msgid "Are you sure you want to delete?"
+msgstr "Weet u zeker dat u wilt doorgaan met verwijderen?"
+
+msgid "This action cannot be undone"
+msgstr "Deze actie kan niet ongedaan worden gemaakt."
+
+msgid "Confirm"
+msgstr "Bevestigen"
+
+msgid "Cancel"
+msgstr "Annuleren"
+
+msgid "Log Detail"
+msgstr "Logboek bekijken"
+
+msgid "Ok"
+msgstr "Oké"
+
+msgid "Details"
+msgstr "Details"
+
+msgid "Rclone configuration file path"
+msgstr "Rclone-configuratielocatie"
+
+msgid "Base mount path"
+msgstr "Hoofd-aankoppellocatie"
+
+msgid "Path where the remote disks are mounted"
+msgstr "De locatie waar externe schijven worden aangekoppeld"
+
+msgid "Filenames to be ignored"
+msgstr "Te negeren bestandsnamen"
+
+msgid "Comma separed filenames to be ignored by inotify"
+msgstr ""
+"Een kommagescheiden lijst met bestandsnamen die dienen te worden genegeerd"
+" door inotify"
+
+msgid "Terminal command"
+msgstr "Terminalvenster"
+
+msgid ""
```

Comment:
msgid "Command to call a new terminal window in order to perform rclone manager actions"